packages/samples/sudt/src/controllers/sudt.controller.ts

```diff
@@ -0,0 +1,42 @@
+import type { HexString, Hash } from '@ckb-lumos/base'
+import { BaseController, Controller, Body, Post } from '@ckb-js/kuai-io'
+import { ActorReference } from '@ckb-js/kuai-models'
+import { BadRequest } from 'http-errors'
+import { SudtModel, appRegistry } from '../actors'
+import { Tx } from '../views/tx.view'
+import { getLock } from '../utils'
+import { SudtResponse } from '../response'
+
+@Controller('sudt')
+export default class SudtController extends BaseController {
+  @Post('/send')
+  async send(
+    @Body() { from, to, amount, typeArgs }: { from: string[]; to: string; amount: HexString; typeArgs: Hash },
```

Comment:
Why is the `from` array here?

Reply:
The account could be a set of addresses when connected to Neuron

Reply:
> The account could be a set of addresses when connected to Neuron

I see.

Reply:
> The account could be a set of addresses when connected to Neuron

So the `from` is array only when the Neuron is connected, right?

Reply:
> > The account could be a set of addresses when connected to Neuron
> 
> So the `from` is an array only when the Neuron is connected, right?

All wallets connected will be considered sets of addresses, even if there is only 1 address bound to the account, e.g. metamask.

So, if the dapp is connected to metamask, the `from` will be `[address_of_metamask]`

Reply:
It can be consider an abstract of account, all accounts connected to the dapp should be sets of addresses